src/components/BlogDetails/BlogDetailsComponents/BlogCommentForm.jsx

```diff
@@ -0,0 +1,62 @@
+"use client";
+
+import React, { useState } from "react";
+
+const BlogCommentForm = () => {
+  const [form, setForm] = useState({ name: "", email: "", comment: "" });
+
+  const handleChange = (e) => {
+    const { name, value } = e.target;
+    setForm({ ...form, [name]: value });
```

Comment:
Please do check for email is valid or not using this helper function **checkEmailForValid** from appHelpers

and also setFrom data like this:  **setForm((prevForm) => ({ ...prevForm, [name]: value }));**